tests/unit/testsuite/Bolt/Boltpay/OrderCreationExceptionTest.php

```diff
@@ -9,9 +9,10 @@
 class Bolt_Boltpay_OrderCreationExceptionTest extends PHPUnit_Framework_TestCase
 {
     /**
-     * Test JSON response for general error with default parameters
+     * @test
+     * JSON response for general error with default parameters
      */
-    public function testGeneralExceptionJson()
+    public function initializeGeneralException_setsCorrectJsonAndHttpCode()
```

Comment:
@adentong 
for all test methods, the format should be as follows:

<methodTested>_<conditionUnderWhichTestIsRun>_<expectedOutcome>

Reply:
Fair point. Let me update.